docs/source/config.json

```diff
@@ -97,10 +97,31 @@
         ]
       }
     },
-    "Metrics & Monitoring": {
+    "Telemetry and Monitoring": {
+      "Exporters": {
+        "Metrics": {
+          "Configuration": "/configuration/metrics",
+          "Prometheus": "/configuration/exporters/prometheus",
+          "OpenTelemetry": "/configuration/exporters/opentelemetry",
+          "Datadog (OTLP)": "/configuration/exporters/datadog-otlp"
+        },
+        "Tracing": "/configuration/tracing",
+        "Logging": "/configuration/logging"
```

Comment:
I think it would be beneficial to have subheading for these sections in the same way as metrics.
Or
Metrics should be combine into one section for consistency,

Reply:
To clarify, you're thinking of:
1 - Creating a new subheading for Tracing and Logging, i.e. the nav would looks like this:

Exporters
- Metrics
   - Config
   - Prometheus
   - Otel
   - Datadog
- New heading
   - Tracing
   - Logging

OR

2 - Combining the four pages under metrics into a single page called metrics, i.e.:

Exporters
- Metrics
- Tracing
- Logging

Is that right?

I would rule out 2, since the config page is already dense. (In fact, it might be nice to make the reference section its own page.)

Though I can see some of the appeal for nesting Tracing and Logging under another subheader, I'm not sure I understand the full argument for it. In the context of "Exporters", "Metrics", "Tracing", and "Logging" seem like siblings, and I'm not sure we need another subheader for the purpose of a more balanced nav.

Reply:
I was thinking 3:

Exporters
* Metrics
  * Config
  * Prometheus
  * Open Telemetry
  * Datadog
  * Honeycomb
* Tracing
  * Config
  * Datadog
  * Honeycomb
  * Open Telemetry
  * Jaeger
  * Zipkin
* Logging
  * Config
  * Stdout
  * File
  * Open Telemetry (future)

Reply:
Ah I see, that makes a lot more sense. Fully in favor!

Reply:
To reduce the amount of nesting, I've removed the "Exporters" top header and renamed each of these subheaders ("Metrics exporters," "Trace exporters," and "Log exporters.") I've kept the file structure intact.

https://github.com/apollographql/router/pull/4129/commits/b423975043a509847644ec3bec1d74692bdae234

LMK if you see some reason to keep the top level "Exporters" header.

Reply:
Looks great!